src/build.ts

```diff
@@ -14,77 +12,82 @@ import { getDirname, getVersion } from "./module-specific.js";
 const version = getVersion();
 const _dirname = getDirname();
 
-const pipe = promisify(pipeline);
-
 export interface Platarch {
   name: string;
   filetype: string;
-};
+}
 
 // These must match the builds that go-tableland is automating
 const platarchs: Platarch[] = [
   {
     name: "darwin-amd64",
-    filetype: ".tar.gz"
+    filetype: ".tar.gz",
   },
   {
     name: "linux-amd64",
-    filetype: ".tar.gz"
+    filetype: ".tar.gz",
   },
   {
     name: "windows-amd64",
-    filetype: ".zip"
-  }
+    filetype: ".zip",
+  },
+  // TODO: add M1 build if/when that makes sense
 ];
 
 const binDirectory = join(_dirname, "..", "..", "bin");
 
-const go = async function () {
-  console.log(`installing validator binaries: ${platarchs.map(pa => `${pa.name}${pa.filetype}`).join(", ")} for version: ${version}`);
+const go = async function (): Promise<void> {
+  console.log(
+    `installing validator binaries: ${platarchs
+      .map((pa) => `${pa.name}${pa.filetype}`)
+      .join(", ")} for version: ${version as string}`
+  );
   for (let i = 0; i < platarchs.length; i++) {
     await fetchAndUnpack(platarchs[i]);
   }
 
   console.log("done installing validator binaries");
 };
 
-const fetchAndUnpack = async function (platarch: Platarch) {
-  const url = `https://github.com/tablelandnetwork/go-tableland/releases/download/v${version}/api-${platarch.name}${platarch.filetype}`
+const fetchAndUnpack = async function (platarch: Platarch): Promise<void> {
+  const url = `https://github.com/tablelandnetwork/go-tableland/releases/download/v${
+    version as string
+  }/api-${platarch.name}${platarch.filetype}`;
   console.log(`fetching: ${url}`);
 
   const res = await fetch(url);
   // TODO: Seems to be a bug in the fetch typings?
-  // @ts-ignore
+  // @ts-expect-error
```

Comment:
Rather than expecting an error, can you use `res.body as unknown as any` here? The expect error is slightly dangerous because it could mask other type errors?

Reply:
Love it, I'll try that now.